src/commands/server/start.ts

```diff
@@ -140,6 +140,10 @@ export default class Start extends Command {
     'skip-version-check': flags.boolean({
       description: 'Skip minimal versions check.',
       default: false
+    }),
+    'skip-cluster-check': flags.boolean({
```

Comment:
`skip-domain-check`

Reply:
`--skip-cluster-availability-check` as we agreed